### ISSUE TITLE
Move definition of steps to steps.py

### ISSUE THREAD

**Issue by mtc2013** [AgileVentures/av-dashboard#1]
While defining them in init works, moving them to steps.py will reduce confusion.

**Comment by brotich** [comment]
PR for the above https://github.com/AgileVentures/av-dashboard/pull/2

**Comment by mtc2013** [comment]
@brotich thanks. you can probably also delete the __init__.py file for now?

**Comment by brotich** [comment]
> you can probably also delete the init.py file for now?

we cannot do this as aloe relies on this file to import  the steps file....see this [test in the aloe framework](https://github.com/aloetesting/aloe/blob/4e79a250db4b8dc1f4a4ef3930491179504efb8b/tests/bad_steps_app/features/steps/__init__.py#L2)

this is also supported by the error below raised when i removed the `__init_.py` on that file:
```
aloe.exceptions.StepDiscoveryError: Cannot load step definition file: 'features/steps.py'
``

**Comment by mtc2013** [comment]
@brotich sounds good.  I have checked that this is working as given.  I will pull it in.  Thanks

**Comment by mtc2013** [comment]
Fixed by #2
